helpers/calendar/ratios_and_utilization.py

```diff
@@ -57,56 +60,79 @@ def get_analytics_ratios_per_room(self, query, start, end):
 
         response = []
         for room in rooms:
-            events_list = []
             checkins = 0
             cancellations = 0
+            calendar_events_list = []
+            app_events_list = []
 
-            checkins, cancellations, events_list = RoomAnalyticsRatios.retrieve_cancellations_and_checkins_for_room(self,  # noqa
+            checkins, cancellations, calendar_events_list, app_events_list = RoomAnalyticsRatios.retrieve_cancellations_and_checkins_for_room(self,  # noqa
```

Comment:
The self here is also unnecessary if you create the object before calling the method.

Reply:
This answer also applies here: https://github.com/andela/mrm_api/pull/210/files#r251803838